package.json

```
@@ -1,63 +1,16 @@
 {
-  "name": "@primer/presentations",
+  "name": "doctocat-template",
+  "repository": "primer/doctocat-template",
```

We should update this repository field so that `Edit on GitHub` links work
```suggestion
  "repository": "primer/presentations",
```